packages/esm-commons-app/src/ward-app/o2-visit-summary-workspace.component.tsx

```diff
@@ -24,10 +24,15 @@ const O2VisitSummaryWorkspace: React.FC<WardPatientWorkspaceProps> = ({ wardPati
     toClass(MATERNAL_ADMISSION_ENCOUNTER_TYPE),
     toClass(POSTPARTUM_DAILY_PROGRESS),
   ];
+  const customJavaScript = `
+    $('#formBreadcrumb').show();
+    $('.simple-form-ui form section').width(300);
+    $('#nav-buttons').hide();
+  `;
```

Comment:
This might work, but I would be a bit worried about using the $ variable.  It is probably safer to use the full `jQuery` instead of `$`